### DISCUSSION TITLE
Citi Hackathon FDC3 Use Case: Create An AppD Reference Implementation

### DISCUSSION THREAD

**Discussion by thetsajeet** [finos/finos-pmcs#335] [Ideas]
- Created a repo
- Developing the application based on the discussion inputs.

[Github repo](https://github.com/Ts-A/AppD-Reference) for the problem statement #331

**Comment by thetsajeet** [comment]
@winmorgan Unable to add the label.

**Reply by winmorgan** [reply]
Label added.
